sklearn/linear_model/ridge.py

```diff
@@ -847,8 +847,10 @@ def fit(self, X, y, sample_weight=None):
         -------
         self : Returns self.
         """
+        alphas = np.asarray(self.alphas)
```

Comment:
ok then I think the last updated version should be ok.  I am using tuples as default value and we expect the users to use np.array.  I think that this should work  well. Unless I am not missing something. In that case explain me :-)